exchanges/coinbasepro/coinbasepro_wrapper.go

```diff
@@ -325,8 +325,12 @@ func (c *CoinbasePro) UpdateAccountInfo(ctx context.Context, assetType asset.Ite
 		return response, err
 	}
 
+	var accountID string
 	var currencies []account.Balance
 	for i := range accountBalance {
+		if accountID == "" {
+			accountID = accountBalance[i].ProfileID
+		}
```

Comment:
This might be the same issue as above, might need to handle different profile ID's to balances which can be independently added to repsonse.Accounts.

Reply:
Yes, applied the changes as well, also on Bitmex.